share/availability/full/reconstruction_test.go

```diff
@@ -10,19 +10,22 @@ import (
 	"github.com/stretchr/testify/require"
 	"golang.org/x/sync/errgroup"
 
-	"github.com/celestiaorg/celestia-node/ipld"
+	"github.com/celestiaorg/celestia-node/share"
+	"github.com/celestiaorg/celestia-node/share/availability/light"
+	availability_test "github.com/celestiaorg/celestia-node/share/availability/test"
+	"github.com/celestiaorg/celestia-node/share/eds"
 )
 
 func init() {
-	ipld.RetrieveQuadrantTimeout = time.Millisecond * 100 // to speed up tests
+	eds.RetrieveQuadrantTimeout = time.Millisecond * 100 // to speed up tests
 }
 
-// TestShareAvailable_OneFullNode asserts that a FullAvailability node can ensure
+// TestShareAvailable_OneFullNode asserts that a Full node can ensure
```

Comment:
```suggestion
// TestShareAvailable_OneFullNode asserts that a full node can ensure
```
nit but we capitalise some and not others